src/strands/experimental/bidi/models/openai.py

```
@@ -557,26 +534,24 @@ async def send(
 
         Args:
             content: Typed event (BidiTextInputEvent, BidiAudioInputEvent, BidiImageInputEvent, or ToolResultEvent).
+
+        Raises:
+            ValueError: If content type not supported (e.g., image content).
         """
-        if not self._require_active():
```

Changes in this method include:
* Raise runtime error if model not "started".
* Raise `ValueError` if content type not supported. It is better to fail fast and explicitly. Users could easily miss warnings and become confused as to why their events weren't processed.